pubspec.lock

```diff
@@ -334,6 +334,14 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "0.17.3"
+  cupertino_icons:
```

Comment:
Ideally, all the `pubspec.lock` changes should be reverted unless there was an explicit update to the packages!